### DISCUSSION TITLE
Traffmonetizer has discontinued the $5 bonus for new accounts.

### DISCUSSION THREAD

**Discussion by hieunguyen8794** [engageub/InternetIncome#105] [General]
Currently, is anyone else experiencing what I am? I've been referring people to Traffmonetizer, but recently, new accounts no longer receive the $5 bonus, and now nobody wants to use this app anymore. Normally, it offers the lowest rate at $1 for 10GB. It's really disappointing. Is anyone else going through this?

**Comment by engageub** [comment]
Hello,
$5 was only 1 time and that too for new users who sign up and not to the referrer. May be some users were misusing this bonus by creating multiple self referrals to get $5.  
On the long term you get 10% of what they earn.
It would be better to check with Traffmonetizer support for more details on this. 

Thank you

**Comment by hieunguyen8794** [comment]
I understand that, but having your referral get $5, and closer to the withdrawal limit, is still better.
